spec/controllers/provider_management/external_users_controller_spec.rb

```diff
@@ -206,38 +203,34 @@ def post_to_create_external_user_action(options={})
   end
 
   describe "PUT #update_password" do
+    let(:password) { 'password123' }
+    let(:password_confirm) { password }
+    subject { put :update_password, params: { provider_id: provider, id: external_user, external_user: { user_attributes: { password: password, password_confirmation: password_confirm } } } }
 
-    context 'when valid' do
-
-      before(:each) do
-        put :update_password, params: { provider_id: provider, id: external_user, external_user: { user_attributes: { password: 'password123', password_confirmation: 'password123' } } }
-        external_user.reload
-      end
+    before(:each) { travel(-6.months) { external_user } }
 
+    context 'when valid' do
       it 'does not require current password to be successful in updating the user record ' do
-        expect(external_user.user.updated_at).to_not eql frozen_time
+        expect { subject }.to change { external_user.reload.user.updated_at }
       end
 
       it 'redirects to external_user show action' do
+        subject
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3627/files#r538198950)